pkgs/applications/networking/cluster/terraform-providers/default.nix

```diff
@@ -55,6 +55,7 @@ let
 
     # Packages that don't fit the default model
     ansible = callPackage ./ansible {};
+    checkly = callPackage ./checkly {};
```

Comment:
Why doesn't this fit the default model? I don't see anything specific.

Reply:
Not sure, when I tried it did not build.

Reply:
```building
go/src/github.com/checkly/terraform-provider-checkly/checkly/provider.go:7:2: cannot find package "github.com/checkly/checkly-go-sdk" in any of:
        /nix/store/qvxsh2nxjf598mzfi6hl9dy4qv3698x4-go-1.15.6/share/go/src/github.com/checkly/checkly-go-sdk (from $GOROOT)
        /build/go/src/github.com/checkly/checkly-go-sdk (from $GOPATH)
```

I'm not sure how to mitigate that.